Dockerfile.wasi-builder

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest@sha256:4b1d0c4a2d2aaf63b37111f34eb9fa89fa1bf53dd6e4ca954d47caebca4005c2
```

Comment:
+1 to use SHA256 to avoid unexpected changes, but I would add a comment to indicate the tag you chose. If I need to update this value in the future, I will know if I need to pick `latest`, `lts` or any specific tag.

Reply:
Ahh lol! I didn't notice the tag first haha. Sorry for the noise!